addons/dialogue_manager/l10n/zh-cn.po

```diff
@@ -0,0 +1,403 @@
+msgid ""
+msgstr ""
+"Project-Id-Version: Dialogue Manager\n"
+"Language: zh-cn\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+
+msgid "start_a_new_file"
+msgstr "创建新文件"
+
+msgid "open_a_file"
+msgstr "打开已有文件"
+
+msgid "open.open"
+msgstr "打开……"
+
+msgid "open.no_recent_files"
+msgstr "无历史记录"
+
+msgid "open.clear_recent_files"
+msgstr "清空历史记录"
+
+msgid "save_all_files"
+msgstr "保存所有文件"
+
+msgid "test_dialogue"
+msgstr "测试"
+
+msgid "search_for_text"
+msgstr "查找……"
+
+msgid "insert"
+msgstr "插入"
+
+msgid "translations"
+msgstr "翻译"
+
+msgid "settings"
+msgstr "设置"
+
+msgid "show_support"
+msgstr ""
+
+msgid "docs"
+msgstr "文档"
+
+msgid "insert.wave_bbcode"
+msgstr "BBCode [lb]wave[rb]"
+
+msgid "insert.shake_bbcode"
+msgstr "BBCode [lb]wave[rb]"
+
+msgid "insert.typing_pause"
+msgstr "输入间隔"
+
+msgid "insert.typing_speed_change"
+msgstr "输入速度变更"
+
+msgid "insert.auto_advance"
+msgstr "自动切行"
+
+msgid "insert.templates"
+msgstr "模板"
+
+msgid "insert.title"
+msgstr "标题"
+
+msgid "insert.dialogue"
+msgstr "对话引用"
+
+msgid "insert.response"
+msgstr "选项"
+
+msgid "insert.random_lines"
+msgstr "随机行"
+
+msgid "insert.random_text"
+msgstr "随机文本"
+
+msgid "insert.actions"
+msgstr "动作"
+
+msgid "insert.jump"
+msgstr "标题间跳转"
+
+msgid "insert.end_dialogue"
+msgstr "结束对话"
+
+msgid "generate_line_ids"
+msgstr "生成行 ID"
+
+msgid "save_to_csv"
+msgstr "生成 CSV"
+
+msgid "import_from_csv"
+msgstr "从 CSV 导入"
+
+msgid "confirm_close"
+msgstr "是否要保存到“{path}”？"
+
+msgid "confirm_close.save"
+msgstr "保存"
+
+msgid "confirm_close.discard"
+msgstr "不保存"
+
+msgid "buffer.save"
+msgstr "保存"
+
+msgid "buffer.save_as"
+msgstr "另存为……"
+
+msgid "buffer.close"
+msgstr "关闭"
+
+msgid "buffer.close_all"
+msgstr "全部关闭"
+
+msgid "buffer.close_other_files"
+msgstr "关闭其他文件"
+
+msgid "buffer.copy_file_path"
+msgstr "复制文件路径"
+
+msgid "buffer.show_in_filesystem"
+msgstr "在 Godot 侧边栏中显示"
+
+msgid "settings.revert_to_default_test_scene"
+msgstr ""
```

Comment:
This one shows up in the settings window when you specify a custom test scene:

![image](https://github.com/nathanhoad/godot_dialogue_manager/assets/78984/f609f537-bd8b-42e0-9ab9-5776e7060cf7)